common/quotas/dynamic.go

```diff
@@ -120,22 +121,26 @@ func NewMutableRateBurst(
 	rate float64,
 	burst int,
 ) *MutableRateBurstImpl {
-	return &MutableRateBurstImpl{
-		rate:  atomic.NewFloat64(rate),
-		burst: atomic.NewInt64(int64(burst)),
+	d := &MutableRateBurstImpl{
+		rate:  atomic.Uint64{},
+		burst: atomic.Int64{},
 	}
```

Comment:
Do you even need to init them? Why not this?
```suggestion
	d := &MutableRateBurstImpl{}
```

Reply:
That works